experiments/run_finetuning.py

```diff
@@ -302,11 +302,11 @@ def main() -> None:
     )
 
     model = model_adapter.model
-    model = get_peft_model(model, lora_config)
-    model.print_trainable_parameters()
+    lora_model = get_peft_model(model, lora_config)
+    lora_model.print_trainable_parameters()
 
     # create optimizer and scheduler
-    optimizer, lr_scheduler = get_optimizer_and_scheduler(model, finetune_ds["train"], args)
+    optimizer, lr_scheduler = get_optimizer_and_scheduler(lora_model, finetune_ds["train"], args)
```

Comment:
This line is the change that triggered the refactoring. We want to calculate perplexity on fine-tuned model which doesn't have an adapter.